cargo-dist/src/tasks.rs

```diff
@@ -1103,6 +1108,134 @@ impl<'pkg_graph> DistGraphBuilder<'pkg_graph> {
         self.add_global_artifact(to_release, installer_artifact);
     }
 
+    fn add_homebrew_installer(&mut self, to_release: ReleaseIdx) {
+        if !self.global_artifacts_enabled() {
+            return;
+        }
+        let release = self.release(to_release);
+        let release_id = &release.id;
+        let Some(download_url) = &self.inner.artifact_download_url else {
+            warn!("skipping Homebrew formula: couldn't compute a URL to download artifacts from");
+            return;
+        };
+        // TODO ensure this is Homebrew-legal
+        let artifact_name = format!("{release_id}.rb");
+        let artifact_path = self.inner.dist_dir.join(&artifact_name);
+        let hint = format!("brew install {artifact_name}");
```

Comment:
This presumably will need some kind of reference to the tap in the final impl right?

Reply:
Yeah, that's right. For this PR I was focused on getting the formula file rendered out - figured it would make sense to leave the tap stuff to a followup.